cmd/cloudflared/linux_service.go

```diff
@@ -269,6 +272,13 @@ func installSystemd(templateArgs *ServiceTemplateArgs, log *zerolog.Logger) erro
 			return err
 		}
 	}
+	// Create the cloudflared user if it does not exist
+	if err := runCommand("grep", "-q", fmt.Sprintf("^%s:", cloudflaredUser), "/etc/passwd"); err != nil {
```

Comment:
@romeroyonatan probably a best approach would be to use the `id` command.
`id user` would return a non successful exit code if user does not exists, it's probably a better approach than directly reading /etc/passwd.  

On the other hand, `useradd` can be called directly and catch the specific error code if user exists